pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -607,6 +620,52 @@ func (b *BridgePodInterface) createBridge() error {
 	return nil
 }
 
+func (b *BridgePodInterface) switchPodInterfaceWithDummy() error {
+	originalPodInterfaceName := b.podInterfaceName
+	newPodInterfaceName := fmt.Sprintf("%s-nic", originalPodInterfaceName)
+	dummy := &netlink.Dummy{LinkAttrs: netlink.LinkAttrs{Name: originalPodInterfaceName}}
+
+	if b.podNicLink.Type() != "dummy" {
+		// Set arp_ignore=1 on the bridge interface to avoid
+		// the interface being seen by Duplicate Address Detection (DAD).
+		// Without this, some VMs will lose their ip address after a few
+		// minutes.
+		//b.arpIgnoreBridge = true
```

Comment:
Why is this commented out?

Reply:
I want to see the test fail with the expected results before I uncomment.

Reply:
You can verify it locally using a kubevirtci cluster.
For example - https://github.com/kubevirt/kubevirt/tree/master/cluster-up/cluster/k8s-1.18

Do the following from the main kubevirt dir-
```
export KUBEVIRT_PROVIDER=k8s-1.18
export KUBEVIRT_NUM_NODES=2
make cluster-up
make cluster-sync
```

Then focus you DAD test. In the test change the `It` to FIt`.
And run-
`make functest`

Then - 
If you change only the test - `make functest` is enough.
If you change the code `make cluster-sync` is needed again.